examples/trap/trap.description

```diff
@@ -1 +1 @@
-// The trap expression stops a panic and gives access to the error value associated with the panic.
+// The `trap` expression stops a panic and gives access to the error value associated with the panic.
```

Comment:
How about captures instead of stops?
```suggestion
// The `trap` expression captures a panic and gives access to the error value associated with the panic.
```

Reply:
Ah spec term right?